### PR TITLE
Sonar Runner Phase 2

### PR DESCRIPTION
**Pull request by poggdogg**
Built a sonar runner and slaved to Jenkins for code tests.
Code tests produce STDERR right now, still need to work out Jenkinsfile handling.  That will be a different PR

### CODE REVIEW

File: openshift/tools/sonar-scanner.bc.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+kind: Template
```
Comment (james-hollinger) [inline openshift/tools/sonar-scanner.bc.yaml]: Is this a duplicate file?

File: player.sh [hunk 2]
```diff
@@ -200,50 +194,47 @@ function determineMode() {
 
 # Scrubs all PR assets from the environment
 function cleanOcArtifacts() {
-    artifactItems=$(oc get all -n ${PROJECT_PREFIX}-dev | grep -i "\-${BRANCH_NAME}"  | column -t | awk '{print $1}' | sort)
+    artifactItems=$(oc get all,secrets,pvc -n ${PROJECT_PREFIX}-dev | grep -i "\-${BRANCH_NAME}"  | column -t | awk '{print $1}' | sort)
```
Comment (james-hollinger) [inline player.sh]: Glad to see this simplified 👍

File: sonar-scanner/.ssh/id_rsa [hunk 3]
```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (james-hollinger) [inline sonar-scanner/.ssh/id_rsa]: !!!

Comment (james-hollinger) [reply sonar-scanner/.ssh/id_rsa]: should this be here?

File: openshift/tools/sonar-scanner.bc.yaml (hunk 1 above)
Comment (poggdogg) [reply openshift/tools/sonar-scanner.bc.yaml]: Yes.  Deleted

File: sonar-scanner/.ssh/id_rsa (hunk 3 above)
Comment (poggdogg) [reply sonar-scanner/.ssh/id_rsa]: nope, not used either.  It's gone.

**Pull request merged**